protocols/.gitignore

```diff
@@ -1,2 +0,0 @@
-!.gitignore
-
```

Comment:
Why remove the empty protocols directory from versioning?

Reply:
There was an error in protocols loading that was making Kuzzle crash because he was trying to load the `.gitignore` file.  
I fixed the bug and put back the `protocols` directories